diff --git a/packages/flutter_tools/templates/app/pubspec.yaml.tmpl b/packages/flutter_tools/templates/app/pubspec.yaml.tmpl
index 8e9a7d66fb..fa388a4369 100644
--- a/packages/flutter_tools/templates/app/pubspec.yaml.tmpl
+++ b/packages/flutter_tools/templates/app/pubspec.yaml.tmpl
@@ -67,6 +67,11 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.6
 
+  # Tizen-specific dependencies.
+  messageport_tizen: ^0.3.2
+  tizen_app_control: ^0.2.2
+  tizen_app_manager: ^0.2.2
+
 dev_dependencies:
 {{#withPlatformChannelPluginHook}}
   integration_test:
